packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -281,6 +281,32 @@ contract Reserve is IReserve, Ownable, Initializable, UsingRegistry, ReentrancyG
   /*
    * Internal functions
    */
+
+  /**
+   * @notice Computes the ratio of current reserve balance to total stable token valuation.
+   * @return Reserve ratio in a fixed point format.
+   */
+  function getReserveRatio() public view returns (uint256) {
```

Comment:
Lots of duplicated between this and `computeTobinTax`, please call `getReserveRatio` in `computeTobinTax`

Reply:
Will do. I actually got the code to calc the reserve ratio from the tobin tax bit